src/crimson/os/seastore/linked_tree_node.h

```diff
@@ -445,6 +453,9 @@ class ParentNode {
       &raw_children[offset],
       &raw_children[offset + 1],
       (me.get_size() - offset - 1) * sizeof(BaseChildNode<T, node_key_t>*));
+    if (me.get_size() < (children.capacity() / 3 /*should be parameterized*/)) {
+      children.resize(children.capacity() / 2 /*should be parameterized*/);
```

Comment:
Probably worth calling `shrink_to_fit()` at the same time to release the memory ?

Reply:
Ah, yes.